C4G.Unity/Assets/C4G/Editor/Scripts/CodeGenerator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Text;
+
+namespace C4G.Editor
+{
+    public static class CodeGenerator
+    {
+        public static string GenerateDTOClass(ParsedSheet parsedSheet)
+        {
+            var sb = new StringBuilder();
+
+            sb.AppendLine($"public sealed class {parsedSheet.Name}");
+            sb.AppendLine("{");
+
+            foreach (var property in parsedSheet.Properties)
+            {
+                sb.AppendLine($"    public {property.Type} {property.Name} {{ get; set; }}");
```

Comment:
[Question] Maybe we should add spaces calculator?

Reply:
https://github.com/vangogih/C4G/issues/18